src/modules/navigation/InstallListItem.tsx

```diff
@@ -0,0 +1,41 @@
+import Divider from '@material-ui/core/Divider';
+import GetAppIcon from '@material-ui/icons/GetApp';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import { useState } from 'react';
+
+
+const InstallListItem = (props: { key: string }) => {
```

Comment:
Maybe change the name to `InstallPWAListItem` or something that also mentions that this is for the PWA.